src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -180,6 +182,20 @@ onMounted(() => {
     })
 })
 
+onUnmounted(() => {
+    if (!isPristine.value) {
+        activeLayers.value.forEach((layer, index) => {
```

Comment:
you may re-use the ref `layersWithTimestamps` from above, it already filters all the relevant layers

Reply:
this is true, but we commit changes to layers through their indexex for timed layers, which `layersWithTimestamps` can't give.
I could also create a new way to commit changes which gives directly the layer for the change, but I would prefer to keep only one way to alter a timed layer.